frontend/packages/component-library/package.json

```
@@ -294,18 +294,17 @@
     "dist"
   ],
   "scripts": {
-    "build": "NODE_OPTIONS='--max-old-space-size=8192' tsup --dts",
-    "build:fast": "tsup",
+    "build": "tsup --clean",
     "clean": "rimraf dist .turbo",
-    "dev": "yarn build --watch",
-    "dev:fast": "yarn build:fast --watch",
+    "dev": "tsup --watch",
     "lint": "eslint .",
     "lint:fix": "eslint --fix .",
     "prepack": "clean-package",
     "postpack": "clean-package restore",
     "prettier": "prettier --check .",
     "prettier:fix": "prettier --write .",
     "release": "release-it --preRelease=alpha",
+    "start": "yarn dev",
```

```suggestion
    "start": "cd .. && yarn start --filter @code-dot-org/component-library",
```

Just an idea, I'm guessing `tsup --watch` here doesn't end up rolling the build up far enough for apps to see?

---

I think a further improvement would probably be to put `apps` in the `frontend` turborepo, which would then allow it to spin up `yarn start` with all the sub-packages. Part of the issue is that we now have two islands that have some bridges to connect, but really I don't see a reason why `apps` can't be part of `frontend` and thus a top-level `yarn start` in frontend would spin up everything